src/components/Footer.js

```diff
@@ -1,4 +1,5 @@
-import useMyStyles from "../styles/MyStyles";
+import common from "../styles/common";
+import footer from "../styles/footer";
 import Link from "@material-ui/core/Link";
 import Grid from "@material-ui/core/Grid";
 import moment from "moment";
```

Comment:
and btw, moment is no longer supported:
https://momentjs.com/docs/
and it's heavy and big and use `date-fns` for example ;) you can add issue on github "Change moment to other date lib" and then make new PR to do it - exercise to check how it works on github

Reply:
I will create issue then change library as suggested.